README.md

```diff
@@ -7,16 +7,8 @@ This plugin provides client-side integration for the [CodePush service](http://c
 * [Supported Components](#supported-components)
 * [Getting Started](#getting-started)
 * [iOS Setup](#ios-setup)
-    * [Plugin Installation](#plugin-installation-ios)
```

Comment:
wait, why are we taking these out if we never touched those sections in the readme?